app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
+use App\Blog;
+use Illuminate\Support\Facades\Auth;
```

Comment:
You also did not use Auth yet, so no need to add.

Reply:
Now delete it.
Code at the implemented timing